helpers/ed.js

```diff
@@ -52,6 +52,9 @@ ed.makeKeypair = function(hash) {
  * @todo Add description for the params and the return value
  */
 ed.sign = function(hash, privateKey) {
+	if (typeof privateKey === 'string') {
```

Comment:
- So now instead of keeping `Buffer` instance in memory we will always create `Buffer` here every `sign` call? Not very efficient.
- Looks like now we're only passing strings here, condition is not needed. Also after removing the condition some tests should fail, because they still testing behavior with `Buffer`, so that condition is only for tests? 😏

Reply:
Also we should unit test that.

Reply:
Condition is not only for tests. `sodium.crypto_sign_detached` only accepts buffer, so why have to convert it.

Reply:
I mean it should be better to convert it to `Buffer` after creating the keypair, so we will keep `Buffer` instances in memory and just pass them here instead of converting them to `Buffer` every time we call `sign`.